server/routes/changeTeam/handlers/changeTeam.ts

```diff
@@ -32,7 +32,7 @@ export default class ChangeTeamRoutes {
     const probationTeamsWithCount = probationTeams.map(probationTeam => {
       return {
         ...probationTeam,
-        count: teams.find(t => t.teamCode === probationTeam.code)?.count || 0,
+        count: req.route.path.includes('create') ? 0 : teams.find(t => t.teamCode === probationTeam.code)?.count || 0,
```

Comment:
Display count only in the vary tile.

Reply:
![image](https://github.com/ministryofjustice/create-and-vary-a-licence/assets/129746957/c5d722e4-075b-424d-9f38-037a151810b9)
![image](https://github.com/ministryofjustice/create-and-vary-a-licence/assets/129746957/dd684d45-d95c-4e69-8832-d6fc30fbaa41)